data/quarg/quarg news.txt

```diff
@@ -0,0 +1,425 @@
+news "anonymous quarg"
```

Comment:
You're missing a copyright header.

Reply:
Just to clarify, this would be the valid copyright header right:
```
# Copyright (c) 2021 by Michael Zahniser
#
# Endless Sky is free software: you can redistribute it and/or modify it under the
# terms of the GNU General Public License as published by the Free Software
# Foundation, either version 3 of the License, or (at your option) any later version.
#
# Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
# PARTICULAR PURPOSE.  See the GNU General Public License for more details.

```